BO4E/ENUM/Sperrauftragsstatus.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Der Sperrauftragsstatus beschreibt den Status eines <see cref="BO.Auftrag"/>s
+    /// </summary>
+    /// <remarks>Diese Information kann in der EDIFACT-Nachricht des Typs 21039 verwendet werden</remarks>
```

Comment:
Die Z17 und Z30 kommen aus der 21040 oder?
```suggestion
    /// <remarks>Diese Information kann in der EDIFACT-Nachricht des Typs 21039/21040 verwendet werden</remarks>
```